apps/frps/0.61.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   frps:
-    image: snowdreamtech/frps:0.61.0
+    image: snowdreamtech/frps:0.61.1
     container_name: ${CONTAINER_NAME}
     restart: always
     network_mode: host
```

Comment:
The code changes appear to be straightforward and correct. Here's an update for clarity:

```yaml
services:
  frps:
    image: snowdreamtech/frps:0.61.1
    container_name: ${CONTAINER_NAME}
    restart: always
    network_mode: host
```

No irregularities, potential issues, or optimizations were noted in this minimal YAML change.